### ISSUE TITLE
a11y: Fix keyboard navigation and focus display for Global Settings

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#12787]
a11y Issue: 228

The global settings action menu is not accessible with the keyboard.

![image](https://github.com/user-attachments/assets/0b51a378-7705-4ffa-b0b0-aba09d06e647)

This should be fixed by fixing the action menu - need to check this works correctly in this page.

**Comment by IsaSih** [comment]
Tests fail in 
Rancher | v2.11-3d729db8a848fb2523bbc2f3f21aa5367dcdcc69-head
Dashboard | master df18acc
Rancher CLI | v2.11.0-rc.1
The keyboard navigation doesnt access the options inside the action menu as seen in the screenshot. Tried both tab and the arrow keys and none have worked.
Reopening the issue.

https://github.com/user-attachments/assets/017daccf-8288-4495-8b94-55a208c3fa81
https://github.com/user-attachments/assets/7bed66ef-0fa8-464b-a064-d638eab584da

**Comment by aalves08** [comment]
@IsaSih I think this issue was moved forward by mistake... 🤔  @rak-phillip do you agree? Yesterday in our sync we chatted about this and I think your work was still in progress...

If that's the case, our apologies @IsaSih 🙏

**Comment by IsaSih** [comment]
Tests pass in v2.11-1ec5c626b1f615573b7f7269d4eb67ddb8c82749-head
